dev-libs/protobuf/protobuf-3.19.4.ebuild

```diff
@@ -0,0 +1,146 @@
+fi
+
+DESCRIPTION="Google's Protocol Buffers - Extensible mechanism for serializing structured data"
+HOMEPAGE="https://developers.google.com/protocol-buffers/ https://github.com/protocolbuffers/protobuf"
+if [[ "${PV}" == "9999" ]]; then
+	SRC_URI=""
+else
+	SRC_URI="https://github.com/protocolbuffers/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+fi
+
+LICENSE="BSD"
+SLOT="0/30"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~x64-macos"
+IUSE="emacs examples static-libs test zlib"
+RESTRICT="!test? ( test )"
+
+BDEPEND="emacs? ( app-editors/emacs:* )"
+DEPEND="test? ( >=dev-cpp/gtest-1.9[${MULTILIB_USEDEP}] )
+	zlib? ( sys-libs/zlib[${MULTILIB_USEDEP}] )"
+RDEPEND="emacs? ( app-editors/emacs:* )
+	zlib? ( sys-libs/zlib[${MULTILIB_USEDEP}] )"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-3.19.0-disable_no-warning-test.patch"
+	"${FILESDIR}/${PN}-3.19.0-system_libraries.patch"
+	"${FILESDIR}/${PN}-3.16.0-protoc_input_output_files.patch"
+)
+
+DOCS=(CHANGES.txt CONTRIBUTORS.txt README.md)
+
+src_prepare() {
+	default
+
+	# https://github.com/protocolbuffers/protobuf/issues/7413
+	sed -e "/^AC_PROG_CXX_FOR_BUILD$/d" -i configure.ac || die
+
+	# https://github.com/protocolbuffers/protobuf/issues/8082
+	sed -e "/^TEST_F(IoTest, LargeOutput) {$/,/^}$/d" -i src/google/protobuf/io/zero_copy_stream_unittest.cc || die
+
+	# https://github.com/protocolbuffers/protobuf/issues/8459
+	sed \
+		-e "/^TEST(ArenaTest, BlockSizeSmallerThanAllocation) {$/a\\  if (sizeof(void*) == 4) {\n    GTEST_SKIP();\n  }" \
+		-e "/^TEST(ArenaTest, SpaceAllocated_and_Used) {$/a\\  if (sizeof(void*) == 4) {\n    GTEST_SKIP();\n  }" \
+		-i src/google/protobuf/arena_unittest.cc || die
+
+	# https://github.com/protocolbuffers/protobuf/issues/8460
+	sed -e "/^TEST(AnyTest, TestPackFromSerializationExceedsSizeLimit) {$/a\\  if (sizeof(void*) == 4) {\n    GTEST_SKIP();\n  }" -i src/google/protobuf/any_test.cc || die
+
+	# https://github.com/protocolbuffers/protobuf/issues/9392
+	sed -e "s/^AC_PROG_OBJC$/AS_CASE([\$target_os], [darwin*], [AC_PROG_OBJC], [AM_CONDITIONAL([am__fastdepOBJC], [false])])/" -i configure.ac || die
+
+	# https://github.com/protocolbuffers/protobuf/issues/9433
+	sed -e "/^[[:space:]]*static_assert(alignof(T) <= 8, \"\");$/d" -i src/google/protobuf/descriptor.cc || die
+
+	eautoreconf
+}
+
+src_configure() {
+	append-cppflags -DGOOGLE_PROTOBUF_NO_RTTI
+
+	if tc-ld-is-gold; then
+		# https://sourceware.org/bugzilla/show_bug.cgi?id=24527
+		tc-ld-disable-gold
+	fi
+
+	multilib-minimal_src_configure
+}
+
+multilib_src_configure() {
+	local options=(
+		$(use_enable static-libs static)
+		$(use_with zlib)
+	)
+
+	if tc-is-cross-compiler; then
+		# Build system uses protoc when building, so protoc copy runnable on host is needed.
+		mkdir -p "${WORKDIR}/build" || die
+		pushd "${WORKDIR}/build" > /dev/null || die
+		ECONF_SOURCE="${S}" econf_build "${options[@]}"
+		options+=(--with-protoc="$(pwd)/src/protoc")
```

Comment:
```suggestion
		options+=( --with-protoc="$(pwd)/src/protoc" )
```

Reply:
A sample of ebuilds with arrays without unnecessary spaces, e.g. in `media-*/*/*.ebuild`:
(None of them are maintained or written by me)
```
$ grep '=([^ )]' media-*/*/*.ebuild
media-gfx/cura/cura-4.12.1.ebuild:DOCS=(README.md)
media-gfx/cura/cura-4.9.1.ebuild:DOCS=(README.md)
media-gfx/hydrus/hydrus-467.ebuild:     local DOCS=(COPYING README.md Readme.txt help_my_client_will_not_boot.txt db/)
media-gfx/hydrus/hydrus-467.ebuild:     local HTML_DOCS=("${S}"/help/)
media-gfx/hydrus/hydrus-468-r1.ebuild:  local DOCS=(COPYING README.md Readme.txt help_my_client_will_not_boot.txt db/)
media-gfx/hydrus/hydrus-468-r1.ebuild:  local HTML_DOCS=("${S}"/help/)
media-gfx/hydrus/hydrus-469.ebuild:     local DOCS=(COPYING README.md Readme.txt help_my_client_will_not_boot.txt db/)
media-gfx/hydrus/hydrus-469.ebuild:     local HTML_DOCS=("${S}"/help/)
media-gfx/hydrus/hydrus-471.ebuild:     local DOCS=(COPYING README.md Readme.txt help_my_client_will_not_boot.txt db/)
media-gfx/hydrus/hydrus-471.ebuild:     local HTML_DOCS=("${S}"/help/)
media-gfx/hydrus/hydrus-9999.ebuild:    local DOCS=(COPYING README.md Readme.txt help_my_client_will_not_boot.txt db/)
media-gfx/hydrus/hydrus-9999.ebuild:    local HTML_DOCS=("${S}"/help/)
media-gfx/jpeg2ps/jpeg2ps-1.9-r2.ebuild:PATCHES=("${FILESDIR}"/${P}-include.diff)
media-gfx/solvespace/solvespace-3.0.ebuild:             mycmakeargs+=("-DENABLE_OPENMP=OFF")
media-gfx/solvespace/solvespace-3.0.ebuild:             mycmakeargs+=("-DENABLE_OPENMP=ON")
media-libs/SoQt/SoQt-1.6.0.ebuild:DOCS=(AUTHORS ChangeLog HACKING NEWS README)
media-libs/SoXt/SoXt-1.4.0.ebuild:DOCS=(AUTHORS ChangeLog HACKING NEWS README TODO BUGS.txt)
media-libs/hamlib/hamlib-4.3.1.ebuild:DOCS=(AUTHORS NEWS PLAN README README.betatester README.developer)
media-libs/hamlib/hamlib-4.4.ebuild:DOCS=(AUTHORS NEWS PLAN README README.betatester README.developer)
media-libs/imlib/imlib-1.9.15-r5.ebuild:PVP=(${PV//[-\._]/ })
media-libs/mesa/mesa-21.3.5.ebuild:     emesonargs+=(-Dplatforms=${platforms#,})
media-libs/mesa/mesa-21.3.5.ebuild:             emesonargs+=(-Dglvnd=true)
media-libs/mesa/mesa-21.3.5.ebuild:             emesonargs+=(-Dglvnd=false)
media-libs/mesa/mesa-21.3.5.ebuild:                     emesonargs+=($(meson_use d3d9 gallium-nine))
media-libs/mesa/mesa-21.3.5.ebuild:                     emesonargs+=(-Dgallium-nine=false)
media-libs/mesa/mesa-21.3.5.ebuild:                     emesonargs+=($(meson_feature vaapi gallium-va))
media-libs/mesa/mesa-21.3.5.ebuild:                     emesonargs+=(-Dgallium-va=disabled)
media-libs/mesa/mesa-21.3.5.ebuild:                     emesonargs+=($(meson_feature vdpau gallium-vdpau))
media-libs/mesa/mesa-21.3.5.ebuild:                     emesonargs+=(-Dgallium-vdpau=disabled)
media-libs/mesa/mesa-21.3.5.ebuild:                     emesonargs+=($(meson_feature xa gallium-xa))
media-libs/mesa/mesa-21.3.5.ebuild:                     emesonargs+=(-Dgallium-xa=disabled)
media-libs/mesa/mesa-21.3.5.ebuild:                     emesonargs+=($(meson_feature xvmc gallium-xvmc))
media-libs/mesa/mesa-21.3.5.ebuild:                     emesonargs+=(-Dgallium-xvmc=disabled)
media-libs/mesa/mesa-21.3.5.ebuild:     emesonargs+=(-Dvulkan-layers=${vulkan_layers#,})
media-libs/mesa/mesa-21.3.5.ebuild:             DRI_DRIVERS+=("$@")
media-libs/mesa/mesa-21.3.5.ebuild:             GALLIUM_DRIVERS+=("$@")
media-libs/mesa/mesa-21.3.5.ebuild:             VULKAN_DRIVERS+=("$@")
media-libs/mesa/mesa-21.3.6.ebuild:     emesonargs+=(-Dplatforms=${platforms#,})
media-libs/mesa/mesa-21.3.6.ebuild:             emesonargs+=(-Dglvnd=true)
media-libs/mesa/mesa-21.3.6.ebuild:             emesonargs+=(-Dglvnd=false)
media-libs/mesa/mesa-21.3.6.ebuild:                     emesonargs+=($(meson_use d3d9 gallium-nine))
media-libs/mesa/mesa-21.3.6.ebuild:                     emesonargs+=(-Dgallium-nine=false)
media-libs/mesa/mesa-21.3.6.ebuild:                     emesonargs+=($(meson_feature vaapi gallium-va))
media-libs/mesa/mesa-21.3.6.ebuild:                     emesonargs+=(-Dgallium-va=disabled)
media-libs/mesa/mesa-21.3.6.ebuild:                     emesonargs+=($(meson_feature vdpau gallium-vdpau))
media-libs/mesa/mesa-21.3.6.ebuild:                     emesonargs+=(-Dgallium-vdpau=disabled)
media-libs/mesa/mesa-21.3.6.ebuild:                     emesonargs+=($(meson_feature xa gallium-xa))
media-libs/mesa/mesa-21.3.6.ebuild:                     emesonargs+=(-Dgallium-xa=disabled)
media-libs/mesa/mesa-21.3.6.ebuild:                     emesonargs+=($(meson_feature xvmc gallium-xvmc))
media-libs/mesa/mesa-21.3.6.ebuild:                     emesonargs+=(-Dgallium-xvmc=disabled)
media-libs/mesa/mesa-21.3.6.ebuild:     emesonargs+=(-Dvulkan-layers=${vulkan_layers#,})
media-libs/mesa/mesa-21.3.6.ebuild:             DRI_DRIVERS+=("$@")
media-libs/mesa/mesa-21.3.6.ebuild:             GALLIUM_DRIVERS+=("$@")
media-libs/mesa/mesa-21.3.6.ebuild:             VULKAN_DRIVERS+=("$@")
media-libs/mesa/mesa-22.0.0_rc1.ebuild: emesonargs+=(-Dplatforms=${platforms#,})
media-libs/mesa/mesa-22.0.0_rc1.ebuild:         emesonargs+=($(meson_use d3d9 gallium-nine))
media-libs/mesa/mesa-22.0.0_rc1.ebuild:         emesonargs+=(-Dgallium-nine=false)
media-libs/mesa/mesa-22.0.0_rc1.ebuild:         emesonargs+=($(meson_feature vaapi gallium-va))
media-libs/mesa/mesa-22.0.0_rc1.ebuild:         emesonargs+=(-Dgallium-va=disabled)
media-libs/mesa/mesa-22.0.0_rc1.ebuild:         emesonargs+=($(meson_feature vdpau gallium-vdpau))
media-libs/mesa/mesa-22.0.0_rc1.ebuild:         emesonargs+=(-Dgallium-vdpau=disabled)
media-libs/mesa/mesa-22.0.0_rc1.ebuild:         emesonargs+=($(meson_feature xa gallium-xa))
media-libs/mesa/mesa-22.0.0_rc1.ebuild:         emesonargs+=(-Dgallium-xa=disabled)
media-libs/mesa/mesa-22.0.0_rc1.ebuild:         emesonargs+=($(meson_feature xvmc gallium-xvmc))
media-libs/mesa/mesa-22.0.0_rc1.ebuild:         emesonargs+=(-Dgallium-xvmc=disabled)
media-libs/mesa/mesa-22.0.0_rc1.ebuild: emesonargs+=(-Dvulkan-layers=${vulkan_layers#,})
media-libs/mesa/mesa-22.0.0_rc1.ebuild:         GALLIUM_DRIVERS+=("$@")
media-libs/mesa/mesa-22.0.0_rc1.ebuild:         VULKAN_DRIVERS+=("$@")
media-libs/mesa/mesa-22.0.0_rc2.ebuild: emesonargs+=(-Dplatforms=${platforms#,})
media-libs/mesa/mesa-22.0.0_rc2.ebuild:         emesonargs+=($(meson_use d3d9 gallium-nine))
media-libs/mesa/mesa-22.0.0_rc2.ebuild:         emesonargs+=(-Dgallium-nine=false)
media-libs/mesa/mesa-22.0.0_rc2.ebuild:         emesonargs+=($(meson_feature vaapi gallium-va))
media-libs/mesa/mesa-22.0.0_rc2.ebuild:         emesonargs+=(-Dgallium-va=disabled)
media-libs/mesa/mesa-22.0.0_rc2.ebuild:         emesonargs+=($(meson_feature vdpau gallium-vdpau))
media-libs/mesa/mesa-22.0.0_rc2.ebuild:         emesonargs+=(-Dgallium-vdpau=disabled)
media-libs/mesa/mesa-22.0.0_rc2.ebuild:         emesonargs+=($(meson_feature xa gallium-xa))
media-libs/mesa/mesa-22.0.0_rc2.ebuild:         emesonargs+=(-Dgallium-xa=disabled)
media-libs/mesa/mesa-22.0.0_rc2.ebuild:         emesonargs+=($(meson_feature xvmc gallium-xvmc))
media-libs/mesa/mesa-22.0.0_rc2.ebuild:         emesonargs+=(-Dgallium-xvmc=disabled)
media-libs/mesa/mesa-22.0.0_rc2.ebuild: emesonargs+=(-Dvulkan-layers=${vulkan_layers#,})
media-libs/mesa/mesa-22.0.0_rc2.ebuild:         GALLIUM_DRIVERS+=("$@")
media-libs/mesa/mesa-22.0.0_rc2.ebuild:         VULKAN_DRIVERS+=("$@")
media-libs/mesa/mesa-9999.ebuild:       emesonargs+=(-Dplatforms=${platforms#,})
media-libs/mesa/mesa-9999.ebuild:               emesonargs+=($(meson_use d3d9 gallium-nine))
media-libs/mesa/mesa-9999.ebuild:               emesonargs+=(-Dgallium-nine=false)
media-libs/mesa/mesa-9999.ebuild:               emesonargs+=($(meson_feature vaapi gallium-va))
media-libs/mesa/mesa-9999.ebuild:               emesonargs+=(-Dgallium-va=disabled)
media-libs/mesa/mesa-9999.ebuild:               emesonargs+=($(meson_feature vdpau gallium-vdpau))
media-libs/mesa/mesa-9999.ebuild:               emesonargs+=(-Dgallium-vdpau=disabled)
media-libs/mesa/mesa-9999.ebuild:               emesonargs+=($(meson_feature xa gallium-xa))
media-libs/mesa/mesa-9999.ebuild:               emesonargs+=(-Dgallium-xa=disabled)
media-libs/mesa/mesa-9999.ebuild:               emesonargs+=($(meson_feature xvmc gallium-xvmc))
media-libs/mesa/mesa-9999.ebuild:               emesonargs+=(-Dgallium-xvmc=disabled)
media-libs/mesa/mesa-9999.ebuild:       emesonargs+=(-Dvulkan-layers=${vulkan_layers#,})
media-libs/mesa/mesa-9999.ebuild:               GALLIUM_DRIVERS+=("$@")
media-libs/mesa/mesa-9999.ebuild:               VULKAN_DRIVERS+=("$@")
media-libs/openimageio/openimageio-2.2.18.0-r1.ebuild:          use "${cpufeature%:*}" && mysimd+=("${cpufeature#*:}")
media-libs/openimageio/openimageio-2.2.18.0-r1.ebuild:  [[ -z ${mysimd} ]] && mysimd=("0")
media-libs/openimageio/openimageio-2.2.19.0.ebuild:             use "${cpufeature%:*}" && mysimd+=("${cpufeature#*:}")
media-libs/openimageio/openimageio-2.2.19.0.ebuild:     [[ -z ${mysimd} ]] && mysimd=("0")
media-libs/openimageio/openimageio-2.3.11.0-r2.ebuild:          use "${cpufeature%:*}" && mysimd+=("${cpufeature#*:}")
media-libs/openimageio/openimageio-2.3.11.0-r2.ebuild:  [[ -z ${mysimd} ]] && mysimd=("0")
media-libs/openimageio/openimageio-2.3.11.0.ebuild:             use "${cpufeature%:*}" && mysimd+=("${cpufeature#*:}")
media-libs/openimageio/openimageio-2.3.11.0.ebuild:     [[ -z ${mysimd} ]] && mysimd=("0")
media-libs/openimageio/openimageio-2.3.12.0-r2.ebuild:          use "${cpufeature%:*}" && mysimd+=("${cpufeature#*:}")
media-libs/openimageio/openimageio-2.3.12.0-r2.ebuild:  [[ -z ${mysimd} ]] && mysimd=("0")
media-libs/osl/osl-1.11.16.0-r2.ebuild:         use "${cpufeature%:*}" && mysimd+=("${cpufeature#*:}")
media-libs/osl/osl-1.11.16.0-r2.ebuild: [[ -z ${mysimd} ]] && mysimd=("0")
media-libs/osl/osl-1.11.17.0-r2.ebuild:         use "${cpufeature%:*}" && mysimd+=("${cpufeature#*:}")
media-libs/osl/osl-1.11.17.0-r2.ebuild: [[ -z ${mysimd} ]] && mysimd=("0")
media-libs/osl/osl-1.12.4.1-r1.ebuild:          use "${cpufeature%:*}" && mysimd+=("${cpufeature#*:}")
media-libs/osl/osl-1.12.4.1-r1.ebuild:  [[ -z ${mysimd} ]] && mysimd=("0")
media-libs/portmidi/portmidi-234.ebuild:                mycmakeargs+=(-DJAR_INSTALL_DIR="${EPREFIX}/usr/share/${PN}/lib")
media-libs/quarter/quarter-1.1.0.ebuild:DOCS=(AUTHORS ChangeLog NEWS README)
media-libs/simage/simage-1.8.0-r1.ebuild:DOCS=(AUTHORS ChangeLog NEWS README)
media-libs/simage/simage-1.8.1.ebuild:DOCS=(AUTHORS ChangeLog NEWS README)
media-libs/simage/simage-9999.ebuild:DOCS=(AUTHORS ChangeLog NEWS README)
media-libs/x264/x264-0.0.20190903-r1.ebuild:PATCHES=("${FILESDIR}"/${P}-STRINGS.patch)
media-libs/zimg/zimg-2.9.2.ebuild:PATCHES=("${FILESDIR}"/${PN}-2.9.2-gcc-10.patch)
media-plugins/grilo-plugins/grilo-plugins-0.3.14.ebuild:                emesonargs+=($(meson_feature gnome-online-accounts goa))
media-plugins/grilo-plugins/grilo-plugins-0.3.14.ebuild:                emesonargs+=(-Dgoa=disabled)
media-plugins/vdr-cinebars/vdr-cinebars-0.0.5.ebuild:PATCHES=("${FILESDIR}/${P}_makefile.diff")
media-plugins/vdr-mp3ng/vdr-mp3ng-0.0.1_pre5-r5.ebuild:DOCS=(HISTORY MANUAL README README-MORONIMO examples/network.sh.example)
media-plugins/vdr-weatherng/vdr-weatherng-0.0.8_pre3-r1.ebuild:PATCHES=("${FILESDIR}/${P}-i18n-fix.diff")
media-radio/flrig/flrig-1.4.2.ebuild:DOCS=(AUTHORS ChangeLog README)
media-radio/flrig/flrig-1.4.3.ebuild:DOCS=(AUTHORS ChangeLog README)
media-radio/flrig/flrig-1.4.4.ebuild:DOCS=(AUTHORS ChangeLog README)
media-sound/bpmdetect/bpmdetect-0.7.0_pre20170810.ebuild:PATCHES=("${FILESDIR}"/${P}-gcc-11.patch)
media-sound/cadence/cadence-0.9.1.ebuild:       myemakeargs=(PREFIX="${EPREFIX}/usr"
media-sound/cadence/cadence-9999.ebuild:        myemakeargs=(PREFIX="${EPREFIX}/usr"
media-sound/puddletag/puddletag-2.0.1-r1.ebuild:DOCS=(changelog NEWS THANKS TODO)
media-sound/puddletag/puddletag-2.1.0.ebuild:DOCS=(changelog NEWS THANKS TODO)
media-sound/puddletag/puddletag-2.1.1.ebuild:DOCS=(changelog NEWS THANKS TODO)
media-tv/mythtv/mythtv-31.0-r6.ebuild:  myconf+=(--prefix="${EPREFIX}"/usr)
media-tv/mythtv/mythtv-31.0-r6.ebuild:  myconf+=(--libdir="${EPREFIX}"/usr/$(get_libdir))
media-tv/mythtv/mythtv-31.0-r6.ebuild:  myconf+=(--libdir-name=$(get_libdir))
media-tv/mythtv/mythtv-31.0-r6.ebuild:  myconf+=(--mandir="${EPREFIX}"/usr/share/man)
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--compile-type=debug)
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--disable-stripping) # FIXME: does not disable for all files, only for some
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--enable-valgrind) # disables timeouts for valgrind memory debugging
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--compile-type=release)
media-tv/mythtv/mythtv-31.0-r6.ebuild:  has ccache "${FEATURES}" || myconf+=(--disable-ccache)
media-tv/mythtv/mythtv-31.0-r6.ebuild:  has distcc "${FEATURES}" || myconf+=(--disable-distcc)
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--cpu="${i}")
media-tv/mythtv/mythtv-31.0-r6.ebuild:  myconf+=($(use_enable altivec))
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--with-bindings=perl,python)
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--without-bindings=python)
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--with-bindings=perl)
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--without-bindings=perl)
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--with-bindings=python)
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--without-bindings=perl,python)
media-tv/mythtv/mythtv-31.0-r6.ebuild:  use python && myconf+=(--python="${EPYTHON}")
media-tv/mythtv/mythtv-31.0-r6.ebuild:  myconf+=($(use_enable java bdjava))
media-tv/mythtv/mythtv-31.0-r6.ebuild:  myconf+=(--enable-symbol-visibility)
media-tv/mythtv/mythtv-31.0-r6.ebuild:  myconf+=(--enable-pic)
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--enable-cross-compile --arch=$(tc-arch-kernel))
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--cross-prefix="${CHOST}"-)
media-tv/mythtv/mythtv-31.0-r6.ebuild:  myconf+=($(use_enable bluray libbluray_external))
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:        myconf+=(--prefix="${EPREFIX}"/usr)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:        myconf+=(--libdir="${EPREFIX}"/usr/$(get_libdir))
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:        myconf+=(--libdir-name=$(get_libdir))
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:        myconf+=(--mandir="${EPREFIX}"/usr/share/man)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--compile-type=debug)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--disable-stripping) # FIXME: does not disable for all files, only for some
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--enable-valgrind) # disables timeouts for valgrind memory debugging
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--compile-type=release)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:        has ccache "${FEATURES}" || myconf+=(--disable-ccache)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:        has distcc "${FEATURES}" || myconf+=(--disable-distcc)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--cpu="${i}")
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:        myconf+=($(use_enable altivec))
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--with-bindings=perl,python)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--without-bindings=python)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--with-bindings=perl)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--without-bindings=perl)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--with-bindings=python)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--without-bindings=perl,python)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:        use python && myconf+=(--python="${EPYTHON}")
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:        myconf+=($(use_enable java bdjava))
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:        myconf+=(--enable-symbol-visibility)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:        myconf+=(--enable-pic)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--enable-cross-compile --arch=$(tc-arch-kernel))
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--cross-prefix="${CHOST}"-)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:        myconf+=($(use_enable bluray libbluray_external))
media-video/atomicparsley-wez/atomicparsley-wez-0.9.6.ebuild:DOCS=(Changes.txt CREDITS)
media-video/lsdvd/lsdvd-0.17.ebuild:DOCS=(AUTHORS README ChangeLog)
media-video/mpv/mpv-0.33.1-r2.ebuild:   [[ ${PV} != *9999* ]] && mywafargs+=(--disable-build-date)
media-video/mpv/mpv-0.34.0.ebuild:      [[ ${PV} != *9999* ]] && mywafargs+=(--disable-build-date)
media-video/mpv/mpv-9999.ebuild:        [[ ${PV} != *9999* ]] && mywafargs+=(--disable-build-date)
media-video/shotcut/shotcut-21.10.31.ebuild:    use debug || myqmakeargs+=(DEFINES+=NDEBUG)
media-video/shotcut/shotcut-21.12.24.ebuild:    use debug || myqmakeargs+=(DEFINES+=NDEBUG)
media-video/shotcut/shotcut-9999.ebuild:        use debug || myqmakeargs+=(DEFINES+=NDEBUG)
media-video/vcdimager/vcdimager-2.0.1.ebuild:PATCHES=("${FILESDIR}/${P}-pkg-config.patch")
```
This contains e.g. ebuilds such as `media-libs/mesa` which were rewritten for switching from Autotools to Meson and cannot be accused of using old style or any such thing.

Reply:
Again, I make the final call here. Either you change it or this doesn't go in.